test/controllers/BusinessContactDetailsControllerSpec.scala

```diff
@@ -243,6 +249,109 @@ class BusinessContactDetailsControllerSpec extends SpecBase with MockitoSugar wi
           }
         }
 
+        "must save the answer and redirect to the Bank Details page if if bank details are not completed" in {
+
+          val emailVerificationResponse = EmailVerificationResponse(
+            redirectUri = routes.BankDetailsController.onPageLoad().url
+          )
+
+          val mockSessionRepository = mock[AuthenticatedUserAnswersRepository]
+
+          when(mockSessionRepository.set(any())) thenReturn true.toFuture
+
+          when(mockEmailVerificationService.isEmailVerified(
+            emailAddress = any(),
+            credId = any())(any())) thenReturn NotVerified.toFuture
+
+          when(mockEmailVerificationService.createEmailVerificationRequest(
```

Comment:
Can you add a verify for this to ensure we're calling with the correctly worked out `continueUrl`?